lists/python/test_linked_list.py

```diff
@@ -185,9 +185,9 @@ def test_dl_remove_node_that_appears_multiple_times():
 
     assert subject == ['a', 'c', 'b']
 
-########################################
-#     Circularly Linked List Tests     #
-########################################
+# ########################################
+# #     Circularly Linked List Tests     #
```

Comment:
Again looks like some spaces snuck in. I would like to see these removed so the test file remains elegant.